app/services/notifications/moderation/send.rb

```diff
@@ -1,16 +1,19 @@
 # Send notifications from moderation
 module Notifications
   module Moderation
+    MODERATORS_AVAILABILITY_DELAY = 22.hours
```

Comment:
I moved `MODERATORS_AVAILABILITY_DELAY` and `SUPPORTED` to this service as it was being used in this fiel and a related spec.